src/routes/api/playlists/preloaded/+server.ts

```diff
@@ -4,7 +4,10 @@ import type { RequestHandler } from './$types';
 
 export const GET = (async (request) => {
   const playlists = (await readPath<Database.Playlists.Preloaded>('/playlists/preloaded')) || {};
-  const playlistArray = Object.entries(playlists).map(([key, val]) => val);
+  const playlistArray = Object.entries(playlists).map(([key, val]) => ({
+    ...val,
+    words: val.words?.map((word) => word.map((letters) => (letters === false ? null : letters))) ?? [],
+  }));
```

Comment:
Small thing we can improve here to make it simpler to read, using Object.values instead
```suggestion
  const playlistArray = Object.values(playlists).map((playlist) => ({
    ...playlist,
    words: playlist.words?.map((word) => word.map((letters) => (letters === false ? null : letters))) ?? [],
  }));
```

Reply:
can do!